src/utils/Multicall.ts

```diff
@@ -16,6 +17,7 @@ export type Call3 = {
 };
 
 const DETERMINISTIC_MULTICALL_ADDRESS = "0xcA11bde05977b3631167028862bE2a173976CA11";
+const MULTICALL_DEPLOYER = "0x05f32b3cc3888453ff71b01135b34ff8e41263f2";
```

Comment:
I missed making this a checksum address in the first pass.

```suggestion
const MULTICALL_DEPLOYER = "0x05f32B3cC3888453ff71B01135B34FF8e41263F2";
```